app/pages/project/home/project-home.jsx

```diff
@@ -29,6 +32,13 @@ const ProjectHomePage = (props) => {
     };
   }
 
+  // variant div: true shows the workflow buttons below
+  // and is for users in a split variant group that allows workflow promotion
+  // so here, we show the get started link if div: false
+  // since users who are not being promoted will work on only one workflow.
+  const showGetStartedLink = (!props.showWorkflowButtons && projectIsNotRedirected) ||
+    (projectIsNotRedirected && props.splits && props.splits['workflow.assignment'] && !props.splits['workflow.assignment'].variant.value.div)
```

Comment:
The logic is a bit confusing here. Why isn't `props.showWorkflowButtons` set to true if the split variant shows the workflow buttons?

Reply:
I'll have to think about this. `props.showWorkflowButtons` is set to true if the project is configured to use workflow assignment or if they have `user_chooses_workflow` enabled. I could check for the split there, however, I wanted to use the `VisibilitySplit` component from the Seven-Ten-Client and have the visibility logic separate from the logic that is independent of an experiment running. I can see how it's confusing though here. 

I think the solution I'll do is also add a boolean to the split variant for the visibility split of the get started link and wrap the `Link` component also with `VisibilitySplit` so it's more clear what is going on.